files/setup_bastion.yaml

```diff
@@ -42,6 +42,17 @@
     net2Routes: __net2_routes__
     extraGateway: __extra_gateway__
   tasks:
+
+    - name: Check if stack update or create and register variable
+      shell: if [[ $(openstack stack show "openshift-{{ osTenantName }}" -f value -c stack_status \
+                --os-auth-url "{{ osAuthUrl }}" \
+                --os-username "{{ openstackOpenshiftUsername }}" \
+                --os-project-name "{{ osTenantName }}" \
+                --os-password "{{ openstackOpenshiftPassword }}") = *CREATE* ]] ; then echo "True" ; else echo "False" ; fi
+      register: create_value
```

Comment:
I don't like this variable name - its not clear enough what its for. Try something more descriptive.

Reply:
Like stack status for example

Reply:
Could this be changed so that it returns a Boolean value as opposed to echoing a string?